scripts/csv-importer.py

```diff
@@ -0,0 +1,259 @@
+import csv
```

Comment:
Having used the script, I have some observations as well.
1. if `input("CSV file path")` was argparse.arg then one would be able to use tab to fill file location
2. csv from our excel has `columns[0]=='L.p.'` I believe the script should by default to use `colums[1]` which is Nazwa in our case.
3.
```bash
Default item name will use the value in column: L.p..
Use a different item name? (y/n) y
Item name: Nazwa
```
I propose to change "Item name" to sth less misleading - it's not item name, it's column name, which items' names will be taken from.
4. `print("Enter names of fields that should be made private, separated by commas (only commas, not comma+space!).")`
solution can be lstrip() before. I think sth like that should work:
```python
private_fields = list(map(str.lstrip, input(">>").split(",")))
```